server/src/views/frame.js

```diff
@@ -199,7 +199,7 @@ class TimeDiff extends React.Component {
       d = new Date(d);
     }
     let months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
-    let month = months[d.getMonth() - 1];
+    let month = months[d.getMonth()];
```

Comment:
FWIW, it's possibly better to use something like `toLocaleString()` -- if we know the user's preferred locale (or even default to en-US until we figure out l20n).

```js
var dat = new Date();
console.log(dat.toLocaleString()); // "7/12/2016, 8:35:21 AM"

var options = {
  weekday: 'short',
  day: 'numeric',
  month: 'short',
  year: 'numeric',
  hour: 'numeric',
  minute: 'numeric'
};

console.log(dat.toLocaleString('en-US', options)); // "Tue, Jul 12, 2016, 8:35 AM"
console.log(dat.toLocaleString('en-GB', options)); // "Tue, 12 Jul 2016, 08:35"
console.log(dat.toLocaleString('de', options));    // "Di., 12. Juli 2016, 08:35"
console.log(dat.toLocaleString('es', options));    // "mar., 12 jul. 2016 8:35"
console.log(dat.toLocaleString('fr', options));    // "mar. 12 juil. 2016 à 08:35"
console.log(dat.toLocaleString('it', options));    // "mar 12 lug 2016, 08:35"
console.log(dat.toLocaleString('ru', options));    // "вт, 12 июл. 2016 г., 8:35"
```